templates/collectd.conf.j2

```diff
@@ -37,4 +37,11 @@ LoadPlugin netlink
   QDisc "eth1"
 </Plugin>
 
+LoadPlugin disk
+<Plugin "disk">
+  Disk "sdb"
```

Comment:
These values should be configurable, because not all EC2 instances come with ephemeral drives at these locations.